modules/decision/search_pattern.py

```diff
@@ -0,0 +1,98 @@
+"""
+Initiates/continues a search pattern when the drone cannot find a landing pad.
+"""
+
+from .. import decision_command
+from .. import odometry_and_time
+from math import tan, pi, cos, sin, ceil
+
+class SearchPattern:
+    """
+    Attributes:
+        camera_fov (float): 
+            Camera's field of view, measured in degrees. Use the smallest measurement available.
+            Is essentially assumed to be a circle
+        search_height (float): 
+            The altitude at which the search is conducted. This is used to calculate the pattern.
+            It does not make the drone go to this height
+        search_overlap (float): 
+            Overlap between passes, between 0 and 1. Recomended probably ~0.5, experiment with less.
+        current_position (OdometryAndTime):
+            The drone's current position.
+        acceptable_variance_squared (float):
+            The square of the acceptable distance from the target position.
+    """
+
+    def __init__(self,
+                 camera_fov: float,
+                 search_height: float,
+                 search_overlap: float,
+                 current_position: odometry_and_time.OdometryAndTime,
+                 acceptable_variance_squared: float):
```

Comment:
variance refers more to a probability/statistics concept, could you rename this to something like distance_square_threshold?